### DISCUSSION TITLE
How to suggest/contribute the localized translations of this extension?

### DISCUSSION THREAD

**Discussion by hwhsu1231** [microsoft/vscode-cmake-tools#2983] [Q&A] [answered]
There are some localized translations of this extension that I want to suggest/contribute. However, I wonder how should I do so?

* About 1 year ago, I posted an issue in [vscode-loc](https://github.com/microsoft/vscode-loc) to ask this kind of question. And a member of vscode-loc replied me with the following message (https://github.com/microsoft/vscode-loc/issues/894#issuecomment-1160994995):

  > Currently only elements within this repo are addressable. For the ticket attached above, please continue addressing it at that particular repo.

  It seems that we should request the modification of the particular extension in its repo.

* However, according to the comment written [here](https://github.com/microsoft/vscode-cmake-tools/blob/main/i18n/cht/package.i18n.json):

  ```json
  /*---------------------------------------------------------------------------------------------
   *  Copyright (c) Microsoft Corporation. All rights reserved.
   *  Licensed under the MIT License. See License.txt in the project root for license information.
   *--------------------------------------------------------------------------------------------*/
  // Do not edit this file. It is machine generated.
  ```

  It seems that we shouldn't edit files inside `i18n` folder directly.

Therefore, I wonder how to suggest/contribute the localized translations of this extension?

**Comment by elahehrashedi** [comment, marked as answer]
The translations are provided by another team, so we don't need outside contributions for translating English strings. However, if there are some translations to some languages that you think could be improved or they seem to be translated incorrectly, you can report an issue and we will open an issue with the corresponding team to fix the translation.
